File: VB/MasterDetailProject/CustomGridControl/CustomBaseTabControlViewInfo.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraTab
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo
Imports MasterDetailProject.CustomGridControl
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Reflection
Imports System.Text
Imports System.Threading.Tasks

Namespace CustomGridControl
	Friend Class CustomBaseTabControlViewInfo
		Inherits SkinTabControlViewInfo

		Public Sub New(ByVal tab As IXtraTab)
			MyBase.New(tab)
		End Sub

		Public Overrides ReadOnly Property IsMultiLine() As Boolean
			Get
				Return True
			End Get
		End Property

		Public Overrides ReadOnly Property IsAllowMultiLine() As Boolean
			Get
				Return True
			End Get
		End Property

	End Class
End Namespace

File: CS/MasterDetailProject/CustomGridControl/CustomBaseTabControlViewInfo.cs
using DevExpress.XtraTab;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;
using MasterDetailProject.CustomGridControl;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Reflection;
using System.Text;
using System.Threading.Tasks;

namespace CustomGridControl {
    class CustomBaseTabControlViewInfo : SkinTabControlViewInfo {

        public CustomBaseTabControlViewInfo(IXtraTab tab)
            : base(tab) {
        }

        public override bool IsMultiLine {
            get {
                return true;
            }
        }

        public override bool IsAllowMultiLine {
            get {
                return true;
            }
        }

    }
}
